test/policies/proxy/proxy.test.js

```diff
@@ -0,0 +1,174 @@
+const path = require('path');
+const fs = require('fs');
+const request = require('supertest');
+const assert = require('chai').assert;
+const https = require('https');
+const express = require('express');
+const expressApp = express();
```

Comment:
Nip: this can be created in the tests setup rather than at the top of the file.

Reply:
Updating.